src/lib/common/JsonHelper.cpp

```diff
@@ -215,19 +220,38 @@ void JsonHelper::addNumber(const std::string& key, long long value)
   empty = false;
 }
 
+
+
+/* ****************************************************************************
+*
+* JsonHelper::addInt -
+*/
+void JsonHelper::addInt(const std::string& key, long long value)
+{
+  if (!empty)
```

Comment:
This function is exactly equal to addNumber(). Why don't use the existing one?

Reply:
My idea is to remove addNumber in the end.
Number is either Float or Integer, and we have a method `addFloat` and not `addInteger` but instead `addNumber` ... Doesn't make any sense. Confusing,
So, I started this change here, adding the `addInt` method.
Creating an issue about removing `addNumber` in favor of `addInt`

Reply:
So, after last comments in #3058 I understand this PR will be modified to use the existing version of addNumber(long long) and in a next and indepedent PR not related with NGSIv2-regs addFloat(float) will be changed to addNumber(float), isn't it?

Reply:
Fixed in 53980e4

I did a quick test adding a method addNumber(xxx, float).
Got a number of compilation errors, saying that addNumber was ambiguous ...
That problem for another day!